wails-frontend/src/pages/passwordPrompt.tsx

```diff
@@ -52,78 +53,84 @@ function PasswordPrompt() {
     }
   }
 
-  function isDisabled() {
-    if (isNewPassword || isImport) {
-      return !(password && passwordConfirm);
+  const [error, setError] = useState<ErrorObject | null>(null);
+
+  function validate(e: SyntheticEvent) {
+    const form = parseForm(e);
+    const { password, passwordConfirm } = form;
+
+    if (!hasMoreThanFiveChars(password.toString())) {
+      setError({ password: 'Password must have at least 5 characters' });
+      return false;
+    } else if (
+      !hasSamePassword(password.toString(), passwordConfirm.toString())
```

Comment:
Do we really need this casting to `.toString() !? They are already, no!? 🤔

Reply:
Sorry it's mt fault. I will remove them.